Abubakarharuna10.md

```diff
@@ -0,0 +1,41 @@
+---
+name: "Abubakarharuna10"
+
+project: "Artificial Rescuer"
+
+Slack Username "@abubakarharuna"
+
+---
```

Comment:
Small thing: could you edit this header section to match the format of the example? 

It should follow this format:

```
---
name: "@Abubakarharuna10"
project: "Artificial Rescuer"
---
```

Reply:
Sure!